### PR TITLE
Rancher Desktop instructions

### PR DESCRIPTION
**Pull request by udf2457**
Ref. #1722

### CODE REVIEW

**Comment by eddumelendez** [comment]
Thanks for your contribution, @udf2457! Can you please add the new doc [here](https://github.com/testcontainers/testcontainers-go/blob/main/mkdocs.yml#L104-L105)?

File: docs/system_requirements/rancher.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+2. In Rancher Desktop set `VZ mode` networking.
+3. On macOS CLI (e.g. `Terminal` app), set the following environment variables:
+
+```sh
+export DOCKER_HOST=unix:/$HOME/.rd/docker.sock
+export TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE=/var/run/docker.sock
+export TESTCONTAINERS_HOST_OVERRIDE=$(rdctl shell ip a show vznat | awk '/inet / {sub("/.*",""); print $2}')
+```
+
+As always, remeber that environment variables are not persisted unless you add them to the relevant file for your default shell e.g. `~/.zshrc`.
```
Comment (mdelapenya) [inline docs/system_requirements/rancher.md:24]:
```suggestion
As always, remember that environment variables are not persisted unless you add them to the relevant file for your default shell e.g. `~/.zshrc`.
```

**Review by mdelapenya — APPROVED** [review]
There is only one typo, but other than LGTM, thanks!

**Comment by udf2457** [comment]
Thanks @mdelapenya .  Have applied your typo suggestion. 

There is actually a second typo "requiremetns", I'll fix that in a sec.

**Review by mdelapenya — APPROVED** [review]
LGTM thanks!

**Pull request merged**